en/simulation/README.md

```diff
@@ -274,7 +274,7 @@ There are a number of ways to make the UDP packets available on external interfa
 
 ### Enable MAV_BROADCAST
 
-Enable [MAV_BROADCAST](../advanced_config/parameter_reference.md#MAV_BROADCAST) to broadcast heartbeats on the local network.
+Enable [MAV_BROADCAST](../advanced_config/parameter_reference.md#MAV_BROADCAST) to broadcast heartbeats on the local network. 
```

Comment:
@bkueng @julianoes So MAV_BROADCAST was deleted in https://github.com/PX4/PX4-Autopilot/pull/14460
It looks like the expectation is that this was something needed for Ethernet, and if you need it you start mavlink module with the `-p` flag.

This particular doc is about setting it up for Simulator. I assume we still want this :-). How do we do it/what do we do?

Reply:
Yes you need to add the `-p` flag for SITL if you want that.
For the configurable instances (boards with ethernet), it can be configured with `MAV_i_BROADCAST`

Reply:
I don't think so. Arg ordering does not matter.

Reply:
Thanks @bkueng I have updated this to "Modify Configuration for UDP Broadcast" and moved down near the external broadcast. I guess I could merge the two sections, but I think possibly good to separate the two options.
1. Please check what I wrote here is correct. 
2. Why/why not would you choose to broadcast (p) rather than specific target (t). I.e. do we need to provide a recommendation?

Note module docs out of date?: https://docs.px4.io/master/en/modules/modules_communication.html#usage-2
![image](https://user-images.githubusercontent.com/5368500/121447778-bcf21500-c9d9-11eb-9d2c-ccaa709df17f.png)

Reply:
Both cases are valid: broadcast makes it easy if you're alone in a network, with possible changing IP's. If you have multiple users (simulations running) broadcast leads to chaos.

Reply:
Thanks. Tidied in https://github.com/PX4/PX4-user_guide/pull/1352